src/platform/plugins/shared/data_views/common/data_views/data_views.test.ts

```diff
@@ -434,6 +434,25 @@ describe('IndexPatterns', () => {
     expect(savedObjectsClient.find).toHaveBeenCalledTimes(2);
   });
 
+  test('caches fields toSpec calls', async () => {
```

Comment:
Built on top of this PR: https://github.com/elastic/kibana/pull/224767 to test performance